src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -336,6 +339,22 @@ internal bool ParsedInIterator
             }
         }
 
+        internal bool ParsedInFieldKeywordContext
+        {
+            get
+            {
+                return (this.Flags & NodeFlags.FactoryContextIsInFieldKeywordContext) != 0;
+            }
+        }
+
+        internal bool ParsedInValueKeywordContext
```

Comment:
where is the code for this checked.   there should be some point that is checking this, and thus ensuring that if we do an incremental parse, where a statement used to be in a property, and it moves to be in a method, that the meaning of `field/value` go back to being an identifier (and vice versa).

note: this is not a strange scenario to be in.  people convert properties to methods and back.  So incremental parsing def needs to be good here.

Reply:
Is this property necessary? To switch the context between a property accessor and a method would require changing the containing member definition kind which I suspect invalidates the body of property, accessor, or method.